geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSetCommandsFunctionExecutor.java

```diff
@@ -43,4 +43,10 @@ public long zadd(RedisKey key, List<byte[]> scoresAndMembersToAdd, ZAddOptions o
   public byte[] zscore(RedisKey key, byte[] member) {
     return stripedExecute(key, () -> getRedisSortedSet(key, true).zscore(member));
   }
+
+  @Override
+  public long zrem(RedisKey key, List<byte[]> membersToRemove) {
+    return stripedExecute(key,
+        () -> getRedisSortedSet(key, false).zrem(getRegion(), key, membersToRemove));
```

Comment:
I didn't think about this for ZADD/ZSCORE, but we should also add tests to `AbstractHitsMissesIntegrationTest` verifying that stats should/shouldn't be updated for `ZREM` and all new commands.

Reply:
Added the test coverage.

Reply:
Thank you for adding those tests!